oracle/examples/kintsugi-config.json

```diff
@@ -113,6 +113,27 @@
                     ]
                 ]
             }
+        },
+        {
+            "pair": [
+                "BTC",
+                "VKSM"
+            ],
+            "feeds": {
+                "dia_fair_price": [
+                    [
+                        {
+                            "symbol": "BTC",
+                            "alias": "KBTC"
```

Comment:
The current architecture assumes we use the same price feed for all currency pairs, to avoid overhauling the oracle client I'm assuming it is okay to use the KBTC DIA xLSD price feed for BTC.